src/tox/tox_env/python/pip/pip_install.py

```diff
@@ -166,7 +166,10 @@ def _execute_installer(self, deps: Sequence[Any], of_type: str) -> None:
         outcome.assert_success()
 
     def build_install_cmd(self, args: Sequence[str]) -> list[str]:
-        cmd: Command = self._env.conf["install_command"]
+        try:
+            cmd: Command = self._env.conf["install_command"]
+        except ValueError as exc:
+            raise Fail(f"unable to determine pip install command: {str(exc)}") from exc
```

Comment:
This line needs a test 👍

Reply:
Strange: the following test coverage _does not_ pass (the error isn't raised):

```py
def test_pip_install_empty_command_error(tox_project: ToxProjectCreator) -> None:
    proj = tox_project({"tox.ini": "[testenv]\ninstall_command="})
    result = proj.run("r")
    result.assert_failed()
    assert "unable to determine pip install command" in result.out
```

(...although when I run `python -m tox run` locally, the error does occur and is handled, as expected)

Reply:
Resolved by c80c25c1a494155bb3e2b7ca5617b022017233a9 : my example above didn't invoke `pip install`, so the `install_command` parsing wasn't being attempted during the test.

Reply:
Nope, this is still wrong.  I'll take another look soon.

Reply:
"More resolved" by https://github.com/tox-dev/tox/compare/f1e7cbfb7ad276da81bdc6717fe14d43725571db...217fcd0292252c80fd8ffa04e9df639cb24c3a56